frontend/src/components/worksheets/WorksheetItemList.js

```diff
@@ -200,7 +198,7 @@ class WorksheetItemList extends React.Component {
                     isDummyItem: true,
                     text: '',
                     mode: 'markup_block',
-                    sort_keys: [-1],
+                    sort_keys: [-1], // default sort_key is -1
```

Comment:
Wait so why is this -1 and not 0 @yuqijin ?

Reply:
Because it is the dummy item, which represents the top of the worksheet, and so it is an exception as I mentioned